### PR TITLE
Fix CG by updating strip-ansi from 5.2.0 to 6.0.1

### PR DESCRIPTION
**Pull request by KeYu-dev**
Detail:
Component and governance pipeline detect security vulnerability alert about ansi-regex(v4.1.0).
To update ansi-regex to v5.0.1, I update strip-ansi to v6.0.1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "jest-junit",
-  "version": "12.3.0",
+  "version": "12.3.1",
```
Comment (SimenB) [inline package.json:3]:
```suggestion
  "version": "12.3.0",
```

@palmerj3 will bump this when making the release

**Review by SimenB — APPROVED** [review]
Looks good and CI is happy 🙂

**Review by palmerj3 — APPROVED** [review]

**Comment by SimenB** [comment]
@palmerj3 could you release it as well? 🙂

**Comment by kachkaev** [comment]
@palmerj3 🙏

**Comment by palmerj3** [comment]
Just published 13.0.0 with this

**Pull request merged**